github_release.sh

```diff
@@ -1,6 +1,15 @@
 PACKAGE_VERSION=$(node -p -e "require('./package.json').version")
 REPO=sentry-demos/sentry_react_native
 
+while true; do
+    read -p "Is the right value set for SE tag in .env? Answer y/n: " yn
```

Comment:
@cstavitsky because i accidentally uploaded a apk that had se=will in it. I tried overwriting the release in Github with an updated SE and APK, but it prevented me (it wants a new release altogether).

Reply:
This is a great idea. I suggest we take it a step further and forcibly prevent the release from continuing if the user doesn't have either 1) no local .env file or, 2) has an env file and `SE` is something other than `tda`. I tested the below out briefly and seems to work.

```
# This would go at the top of github_release.sh

 # If you have an env file, and you've set SE variable to something
 # other than tda, prevent the release from happening
 if [ -f ".env" ]
 then
   export $(cat .env | xargs) # Set env vars from .env file so they're accessible
   if [[ $SE && $SE != 'tda' ]]
   then
     printf "Your SE environment variable is set to $SE.\nThe SE variable must be set to tda    (lowercase) to perform a release.\nPlease change it in your env file and try again."
     exit
   fi
 fi

# the rest of the file
```

Reply:
@thinkocapo this is a great idea, thoughts on taking it one step further and forcibly preventing the release unless the SE tag is set to `tda`?

```
# this goes at top of github_release.sh

# If you have an env file, and you've set SE variable to something
 # other than tda, prevent the release from happening
 if [ -f ".env" ]
 then
   export $(cat .env | xargs) # Set env vars from .env file so they're accessible
   if [[ $SE && $SE != 'tda' ]]
   then
     printf "Your SE environment variable is set to $SE.\nThe SE variable must be set to tda    (lowercase) to perform a release.\nPlease change it in your env file and try again."
     exit
   fi
 fi
 
 # rest of file continues...
```

Reply:
Yes that's good it's faster. I'll make the changes and try it out. Thanks. @cstavitsky